### PR TITLE
feat(jest): 79% coverage for IdentityManager

### PR DESCRIPTION
**Pull request by drepram**
**What this PR does** 📖

Reopening #4291 in the hopes of not having identical GitHub Actions error!

**Which issue(s) this PR fixes** 🔨

AP-2128

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Review by stavares843 — CHANGES_REQUESTED** [review]
Captura de ecrã 2022-08-22, às 13 17 10

when running locally this is generating the snapshot, can you commit the snapshot as well?

**Comment by stavares843** [comment]
added the snapshot

**Review by stavares843 — APPROVED** [review]

File: libraries/Iridium/IdentityManager.test.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+describe('Test seedRandom', () => {
+  it('should exist', async () => {
+    const result = await IdentityManager.seedRandom()
+    expect(result).not.toBeFalsy() // The result exists
+  })
+})
+
+describe('Test seedFromWallet', () => {
+  // For some reason, this does not update coverage
```
Comment (stavares843) [inline libraries/Iridium/IdentityManager.test.ts:37]: if it doesn't update the coverage maybe we can ditch the test

File: libraries/Iridium/IdentityManager.test.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      localParam.passphrase,
+      localParam.publicKey,
+    )
+    expect(result).not.toBeFalsy()
+    expect(result).toMatchSnapshot()
+  })
+})
+
+describe('Test seedFromPrivateKey', () => {
+  it('should return an error: Incorrect signature hash size', async () => {
```
Comment (aewing) [inline libraries/Iridium/IdentityManager.test.ts:54]: maybe this could be `it('throws on invalid private key length')`

**Review by aewing — APPROVED** [review]

File: libraries/Iridium/IdentityManager.test.ts (hunk 2 above)
Comment (drepram) [reply libraries/Iridium/IdentityManager.test.ts:54]: Hi! Please check #4435

**Pull request merged**